AABB_tree/include/CGAL/AABB_triangulation_3_triangle_primitive.h

```diff
@@ -103,13 +102,13 @@ namespace CGAL {
         Id& id() { return m_facet; }
 
     private:
-        /// The id, here a C3T3 facet handle
+        /// The id, here a Tr facet handle
```

Comment:
Actually, there is no such "facet handle". That is a `Tr::Facet`, and not a handle.